src/ui/public/timefilter/timefilter.js

```diff
@@ -23,6 +23,12 @@ uiModules
   var Events = Private(EventsProvider);
   var diff = Private(UtilsDiffTimePickerValsProvider);
 
+  config.$bind($rootScope, 'dateFormat:dow', 'dateFormat_dow');
+
+  $rootScope.$watch('dateFormat_dow', function (day) {
```

Comment:
@panda01 Do you have any thoughts on this? I'm not really sure how the timepicker state is managed throughout the app.